lib/uplinkc/uplink_definitions.h

```diff
@@ -6,10 +6,30 @@
 #include <stdio.h>
 #include <string.h>
 
+typedef enum Cipher {
+    STORJ_UNENCRYPTED,
+    STORJ_AESGCM,
+    STORJ_SECRET_BOX,
+    STORJ_INVALID
+} Cipher;
+
+typedef enum CipherSuite {
+    STORJ_ENC_UNSPECIFIED,
+    STORJ_ENC_NULL,
+    STORJ_ENC_AESGCM,
+    STORJ_ENC_SECRET_BOX
```

Comment:
Let's add specific numbers to these...

```
    STORJ_ENC_UNSPECIFIED = 0,
    STORJ_ENC_NULL        = 1,
    STORJ_ENC_AESGCM      = 2,
    STORJ_ENC_SECRET_BOX  = 3
```

Reply:
This is to make it clear that it's just not an enum, but rather a enumeration where the values need to be kept as is.

Reply:
Note to self: we need to do the same on Go side.

Reply:
:+1: